src/library/scala/collection/Map.scala

```diff
@@ -179,9 +179,10 @@ trait MapOps[K, +V, +CC[_, _] <: IterableOps[_, AnyConstr, _], +C]
 
   override /*PartialFunction*/ def applyOrElse[K1 <: K, V1 >: V](x: K1, default: K1 => V1): V1 = getOrElse(x, default(x))
 
-  /** Collects all keys of this map in a set.
-    * @return  a set containing all keys of this map.
-    */
+  /** A set view of the keys contained by this map.
```

Comment:
Return a set view of the keys contained in this map?

Reply:
That's a good question. I prefer minimal words. Instead of, "This class is a map of values," it's enough to say, "A map of values." For an accessor, "Returns a set view" is misleading, because an accessor trivially produces its accessed. The signature of `keys` by contrast ought to be `keys()` with doc, "Returns a collection..." Maybe the rule is that for attributives of the form `X is A`, it's sufficient to document the attribute A. For active verbs, retain the verbiage: "Produce a widget that does A and B."